src/common/components/nav/nav.component.js

```diff
@@ -37,6 +45,34 @@ class NavController{
     this.getNav().subscribe((structure) => {
       this.menuStructure = structure;
     });
+
+    this.subscription = this.sessionService.sessionSubject.subscribe( () => this.sessionChanged() );
+
+    this.templateUrl = this.$window.screen && this.$window.screen.width < 991 ? mobileTemplate.name : desktopTemplate.name;
```

Comment:
This was intended.  If you're on desktop, you get desktop, mobile you get mobile, regardless of window width.  That is why `screen` is used instead of document.